extensions/labgraph_protocol/README.md

```diff
@@ -0,0 +1,33 @@
+# LabGraph Protocol
+
+Provides a framework for creating experiment protocols with labgraph
+
+## Quick Start
+
+### Method 1 - building from source code
+
+**Prerequisites**:
+
+Make sure to install labgraph before proceeding
+
+```
+cd labgraph/extensions/labgraph_protocol
+# HACK: PyQt5-sip has to be installed before PyQt5
+pip install PyQt5-sip==4.19.18
+python setup.py install
```

Comment:
I needed to run the following command: sudo python setup.py install

Reply:
I will update the README to indicate this

Reply:
Actually, I just tried this and didn't need sudo (I was using a virtualenv).
It might depend on which python installation you're using?